services/diversity/config/default.json

```diff
@@ -0,0 +1,16 @@
+{
+    "db": {
+      "host": "localhost",
+      "port": 5432,
+      "user": "user",
+      "password": "password",
+      "database": "database"
+    },
+    "server" : {
+      "port": 6000,
```

Comment:
@cazapatamar had problems with that port number, I wold suggest to use 4003 to keep it simpler and closer to the other services

Reply:
Agree

Reply:
Changed!